src/services/tags/create-tag.test.js

```diff
@@ -0,0 +1,54 @@
+const {expect} = require('chai')
+const { faker } = require('@faker-js/faker')
+const db = require("../../db")
+const createTag = require("./create-tag")
+const { fixtures } = require('../../../test/utils')
+
+
+describe('Create tag', () => {
+	
+	let tag
+	before(async ()=>{
+		  
+		 tag = await createTag({
```

Comment:
kindly have a `createTag` for each test case instead of in the `before` block. 

helps to reason about the flow especially with the error test case we have now. becomes hard to see know where the tag was inserted from

also if the reason is about flakiness due to the `title` possibly being the same, we can change the faker function we're using, maybe `faker.random.words()` for more variety or something better

Reply:
okay, not sure what the articles one is about but just the` tags` is in the scope of this pr so let's deal with that

Reply:
okay then

Reply:
done